index.php

```diff
@@ -8,52 +8,133 @@
-              <small class="help-block with-errors text-danger"></small>
+        <!-- Easiest way to own a home -->
+        <section class="row pb-5">
+          <div class="col-md-6 text-center">
+            <img src="./assets/img/home-love.png" alt="home-love" class="home-love">
+          </div>
+          <div class="col-md-6 text-left easy">
+            <h3 class="easy-head">The easiest way to own a home</h3>
+            <div class="easy-body mt-4 mb-3 row">
+              <p class="col-md-10 pr-4">
+                We make owning a home simpler and achievable.
+                With BALL unique saving plan tailored to you and your financial position,
+                owning a home has never been easier.
+              </p>
             </div>
+            <button class="btn btn-lg btn-get-notified">Get Notified!</button>
+          </div>
+        </section>
+        <!-- /.Easiest way to own a home -->
 
-            <div class="form-group mb-4">
-              <label for="phone"  class="small mb-0">Phone Number <span class="text-muted">(optional)</span> </label>
-              <input type="text" class="form-control col-md-9" placeholder="Phone Number" name="phone">
-              <small class="help-block with-errors text-danger"></small>
+        <!-- Part of a Prestigious Community -->
+        <section class="row mt-5">
+          <div class="col-md-6 text-right easy">
+            <h3 class="easy-head">Be part of a prestigious community</h3>
+            <div class="easy-body mt-4 mb-3 row">
+              <p class="col-md-9 pl-4 offset-md-3 text-right">
+                What’s a home without family. <br> Our community provides the love of a family, from sharing of advice to mentorship and many more.
+              </p>
             </div>
+            <button class="btn btn-lg btn-get-notified">Get Notified!</button>
+          </div>
+          <div class="col-md-6 text-center">
+            <img src="./assets/img/room-like.png" alt="room-like" class="home-love">
+          </div>
+        </section>
+        <!-- /.Part of a Prestigious Community -->
+    </div>
+    <!-- /. Container -->
 
-            <button type="submit" class="btn submit-btn mt-2">Submit</button>
-          </form>
+    <section class="landlord row mt-5 d-none d-md-flex">
+        <div class="col-md-6 be-landlord">
+          <h3 class="landlord-head">Become a landlord</h3>
+          <p class="landlord-text">Join a community of insightful minds <br> where you can own a home with ease.</p>
+          <button class="btn btn-lg landlord-btn">Get Notified!</button>
+        </div>
 
-          <footer class="col-sm-12 mt-5 pt-4 text-center">
-            <small class="text-muted">Strategically build your wealth today.</small>
-          </footer>
+        <div class="col-md-6 text-center key-div">
+          <img src="./assets/img/hand-key.png" alt="hand-key" class="hand-key">
```

Comment:
I think the hand image has been updated in the design. Can you check and update it here please?